2-mandatory/3-financial-times.js

```diff
@@ -23,59 +41,79 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  // TODO
+  let numTitleArr = [];
+  let conditions = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"];
+  for (let title of allArticleTitles) {
+    if (conditions.some((el) => title.includes(el))) {
```

Comment:
this works, but you could use regex so you don't have to type out an array of the conditions :) for example:
```
 if (/\d/.test(title)) {
      numTitleArr.push(title);
 }
```
\d is used to test for numbers present - this is a good site for playing around with regex https://regex101.com/